conda_forge_feedstock_ops/container_utils.py

```diff
@@ -107,13 +110,51 @@ def _get_proxy_mode_container_args():
     ]
 
 
+@dataclass
+class Mount:
+    host_path: Path
+    """
+    The path on the host to mount.
+    """
+    container_path: PurePosixPath
+    """
+    The path in the container to mount to.
+    """
+    read_only: bool = True
+    """
+    Whether to mount the path as read-only.
+    """
+
+    @classmethod
+    def to_cf_feedstock_ops_dir(cls, host_path: Path, read_only: bool = True):
```

Comment:
isn't a `-> Self` missing here?

Reply:
a0b3b461cf6fd3f9c2a5c5550fa29d616dfa4df2
time to add mypy :)